content/kubermatic/master/tutorials_howtos/external_clusters/_index.en.md

```diff
@@ -5,15 +5,16 @@ weight = 7
 
 +++
 
-This section describes how to add and manage existing GKE, AKS, and EKS clusters. Once an external cluster is added and
-registered within the KKP platform, you may then upgrade the control plane version or scale the nodes.
-You can also connect any other clusters in the KKP via kubeconfig. Connected clusters can only be viewed, not edited. 
+This section describes how to add and manage existing Kubernetes clusters called as External Cluster in KKP.
+You can import or connect a cluster.
```

Comment:
Technically we do not support "importing" but connecting, but we use "add" in the UI. "Import" should not be used.